wasm/src/logic/mod.rs

```diff
@@ -133,6 +136,11 @@ impl LogicContext {
             Message::MouseDown(event) => Ok(Some(Event::MouseDown(event))),
             Message::MouseUp(event) => Ok(Some(Event::MouseUp(event))),
             Message::KeyPress(t, code) => Ok(Some(Event::KeyPress(t as u16, code))),
+            Message::AudioLoaded(id) => {
+                let mut res = crate::communication::RESOURCE_TABLE.write();
+                res.store(rask_engine::resources::Sound {}, id as usize);
```

Comment:
I can't replace it with () because our getstore resource table would get messed up, tried that. Also thought of removing the braces, just forgot to in the end